parsons/databases/database/constants.py

```diff
@@ -153,11 +153,9 @@
 
 VARCHAR = "varchar"
 FLOAT = "float"
-
-DO_PARSE_BOOLS = False
 BOOL = "bool"
-TRUE_VALS = ("TRUE", "T", "YES", "Y", "1", 1)
```

Comment:
wowza

Reply:
Oh yeah I should delete those lines lol, they're not used anymore